spec/features/idv/in_person_spec.rb

```diff
@@ -290,7 +291,7 @@
     end
 
     it 'shows the address page' do
-      complete_state_id_step(user, same_address_as_id: false)
+      complete_state_id_controller(user, same_address_as_id: false)
```

Comment:
I was thinking about why we changed the name here but the other methods below (ie: complete_address step, complete_ssn_step) are not renamed to _controller. I think we used the original method and added an if/else block with the flag. You don't need to do that. (That would involve changing 50/50 tests and more work. Keep it as is- just an observation or pointing it out if others have the same thought.)